repos/system_upgrade/el7toel8/actors/authselectscanner/libraries/library.py

```diff
@@ -0,0 +1,267 @@
+import ConfigParser
+import os
+import re
+import StringIO
+import textwrap
+
+from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.models import Authselect
+
+
+def read_file(config):
```

Comment:
I see identical function in libraries/pam.py
Can this code be shared?

Reply:
I'd rather leave it as is TBH and leave room for improvement in other ways later.

Reply:
Leaving this as is.